odiglet/pkg/kube/runtime_details/inspection.go

```diff
@@ -176,7 +183,91 @@ func runtimeInspection(pods []corev1.Pod, ignoredContainers []string) ([]odigosv
 	return results, nil
 }
 
+// updateRuntimeDetailsWithDockerFileEnvs checks if relevant environment variables are set in the Dockerfile
+// and updates the RuntimeDetailsByContainer accordingly.
+func updateRuntimeDetailsWithDockerFileEnvs(ctx context.Context, criClient criwrapper.CriClient, pod corev1.Pod, container corev1.Container,
+	programLanguageDetails common.ProgramLanguageDetails, resultsMap *map[string]odigosv1.RuntimeDetailsByContainer) {
+	// Retrieve environment variable names for the specified language
+	envVarNames, exists := envOverwrite.EnvVarsForLanguage[programLanguageDetails.Language]
+	if !exists {
+		return
+	}
+
+	// Verify if environment variables already exist in the container manifest.
```

Comment:
Why not always collect the relevant env vars from Dokcerfile?
I don't have a good scenario in my head, but my feeling is that the more stateless we can make everything the simpler it will be